profiles/sssd/README

```diff
@@ -1,26 +1,24 @@
-Enable SSSD for system authentication (also for local users only)
-=================================================================
+Enable SSSD for system authentication (optionally also for local users)
+=======================================================================
```

Comment:
I would keep the original sentence. The reason is that this profile can be used for both SSSD users and local users regardless on the files provider configuration. We don't have any other profile that should be used for local users only mode.